Pipeline/RunIDZCodeReview/RunCodeReview.groovy

```diff
@@ -132,15 +132,15 @@ else
 		codeRev.getAllDDNames().each({ ddName ->
 			if (ddName == 'XML') {
 				def file = new File("${properties.workDir}/CodeReview${ddName}.xml")
-				codeRev.saveOutput(ddName, file, properties.logEncoding)
+				codeRev.saveOutput(ddName, file, properties.logEncoding/*, true*/)
```

Comment:
@jbyibm
I think the new saveOutput method will break backward compatibility. While the new method will allow to trim and remove the ASA control characters from the JCL when saving the DD output, can we indicate the required DBB toolkit level?

Reply:
Updating DBB to the last PTF can be delayed in customer organizations. In my last commit I print this message if the new API is not available.
```
*? Warning the output file /var/jenkins/agent/genapp/workspace/GenAppNazarePipelineZunit_master/BUILD-211/build.20201217.032844.028/CodeReviewJUNIT.xml
*? will have an extra space at the beginning of each line.
*? Updating DBB to the latest PTF with ASA control characters API for JCLExec is highly recommended.
```